python/setup.py.in

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python3
+
+"""
+Setup for SWIG Python bindings for lex-tools
+"""
+from os import path
+from distutils.core import Extension, setup
+from distutils.command.build import build
+
+
+class CustomBuild(build):
+    sub_commands = [
+        ('build_ext', build.has_ext_modules),
+        ('build_py', build.has_pure_modules),
+        ('build_clib', build.has_c_libraries),
+        ('build_scripts', build.has_scripts),
+    ]
+
+
+def get_sources():
+    sources = ['lex_tools.i']
+    cc_sources = ['lrx_processor.cc']
+    rel_path = '..'
+    sources.extend(path.join(rel_path, f) for f in cc_sources)
+    return sources
+
+def get_include_dirs():
+    # get rid of '-I' in LTTOOLBOX_CFLAGS and split list from 1, to remove extra element ''
+    # Strip extra space at end
+    dirs = '@LTTOOLBOX_CFLAGS@'.split('-I')[1:]
```

Comment:
This seems really sketchy. What does `'@LTTOOLBOX_CFLAGS@'` actually look like? There should be a simpler way to do this.

Reply:
That is sketchy. The order of CFLAGS is not defined and there might not even be a `-I`. This should probably be a conditional regex capture instead. But it'll 99% work just fine for our builds.

Reply:
> This seems really sketchy.

`.replace('-I', '').split()` is a better solution

> What does `'@LTTOOLBOX_CFLAGS@'` actually look like?

`'-I/usr/local/include/lttoolbox-3.5 -I/usr/local/lib/lttoolbox-3.5/include -I/usr/include/libxml2 '`

I would need to use these flags in core apertium as well. Should I switch to regex?

Reply:
`.replace('-I', '').split()` is good enough for me.